airbyte-cdk/python/unit_tests/connector_builder/test_connector_builder_handler.py

```diff
@@ -657,10 +657,22 @@ def test_create_source():
 
     source = create_source(config, limits)
 
+    max_retries = 0
```

Comment:
Is it safe enough to rely on `assert http_client._disable_retries`? Else, it feels like we are copy/pasting the logic of 
`HttpClient. _max_retries` which means potentially duplication of work every time the logic change as this will need to be updated

Reply:
Good catch. I added in that logic before consolidating it in the http_client._max_retries method.

Reply:
Honestly I don't think this assertion is even worth keeping around since it's already tested via http_client.

Reply:
I don't think this is enough. We need to make sure the Connector Builder disable retries too

Reply:
Yeah, fair point. Added a separate test to specifically test for this.